sites/friday/src/manage-resources/_index.md

```diff
@@ -0,0 +1,279 @@
+
+{{% vendor/name %}} allows you to configure resources (CPU, RAM, disk) per environment for each of your applications and services.
+You can also add instances for each of your applications depending on your needs.
+
+For example, you can scale vertically and allocate more resources to your production and staging environments
+than to your development environments.
+This flexibility allows you to optimize performance and costs.
+
+You can even scale horizontally if your applications are struggling with high load, or if you're expecting a traffic spike,
+by adding more instances for your applications and workers.
+
+{{< note >}}
+
+When you first deploy your {{% vendor/name %}} project, and whenever you add a new application or service,
+you get notified that you need to configure the resources on your environment:
+
+```bash
+The push completed but resources must be configured before deployment can succeed.
+```
+
+This is because {{% vendor/name %}} doesn't know the exact amount of resources your project needs to run smoothly.
+Therefore, your app can only be successfully deployed after you've configured those resources
+through the {{% vendor/name %}} Console or CLI.
+
+For better guidance on how to configure resources,
+use `{{% vendor/cli %}} push` instead of `git push`.
+
+{{< /note >}}
+
+## Configure resources
+
+### Vertical scaling
+
+You can define how much CPU, RAM, and disk storage you want to allocate to each individual container.
+To do so, follow these steps:
+
+{{< codetabs >}}
+
++++
+title= Using the CLI
++++
+
+Run the `{{% vendor/cli %}} resources:set` command, and follow the prompts to set resources for each of your apps and services.
+
+{{< note title= "Tip" >}}
+
+For further guidance on how to set resources using the CLI, run the `{{% vendor/cli %}} resources:set --help` command.
+
+{{< /note >}}
+
+After you've set resources, your environment is redeployed,
+which causes a short downtime.
+
+<--->
+
++++
+title= From the Console
++++
+
+1. Navigate to your environment.
+2. Select an app or service in the tree on the left-hand side:
+   ![Apps and services tree](/images/flexible-resources/apps-services-tree.png "0.2")
+   A window pops up.
+3. Click **Configure**.
+4. For each app and service, select a CPU & RAM combination, and enter the amount of disk/storage you want to allocate.
+   ![Configure your resources on the current environment window](/images/flexible-resources/configure-flexible-resources.png)
+   Note that the values available in the **CPU & RAM** menus depend on the [container profile](#advanced-container-profiles) of each instance.
+5. Click **Save**.</br>
+   You environment is redeployed, which causes a short downtime.
+
+{{< /codetabs >}}
+
+### Horizontal scaling
+
+For apps and workers, you can also define how many instances you want to deploy.
+To do so, follow these steps:
+
+{{< codetabs >}}
+
++++
+title= Using the CLI
++++
+
+To define how may instances of an app or worker you want to deploy,
```

Comment:
Small typo here, I believe `may` should be `many`.

Reply:
Thanks!